drivers/hmis/app/models/hmis/hud/custom_service_category.rb

```diff
@@ -18,6 +18,15 @@ class Hmis::Hud::CustomServiceCategory < Hmis::Hud::Base
 
   validates_presence_of :name, allow_blank: false
 
+  scope :non_hud, -> do
+    hud_service_category_ids = Hmis::Hud::CustomServiceCategory.
+      left_joins(:service_types).
+      where.not(service_types: { hud_record_type: nil }).
+      distinct
```

Comment:
I think you're missing a `pluck(:id)`?
Apparently Rails is happy with `where(id: scope)` but it doesn't look like that was the intention.

Reply:
Thanks yes 🤦‍♀️